src/azure-cli/azure/cli/command_modules/rdbms/_helptext_mysql.py

```diff
@@ -16,6 +16,15 @@
 type: command
 short-summary: Create a flexible server.
 examples:
+  - name: Create a MySQL flexible server with default params (resource group, location, servername, username, password) with VNET enabled by default.
+    text: |
+        az mysql flexible-server create
```

Comment:
no parameter is required for this command?

Reply:
Yes, we want to make creating step simple and easy so we auto-generate everything if the user doesn't pass along any parameters